### PR TITLE
PEP 656: Clarification on platform definition and musl detection

### PR DESCRIPTION
**Pull request by uranusjr**
According to feedback.

### CODE REVIEW

File: pep-0656.rst [hunk 1]
```diff
@@ -46,6 +46,14 @@ Logic behind the new platform tag largely follows :pep:`600`, and
 require wheels using this tag make similar promises. Please refer to
 the PEP for more details on rationale and reasoning behind the design.
 
+The ``musllinux`` platform tags only apply to Python interpreters
+dynamically linked aginst the musl libc and executed on the runtime
```
Comment (1ace) [inline pep-0656.rst:50]:
```suggestion
dynamically linked against the musl libc and executed on the runtime
```

File: pep-0656.rst [hunk 2]
```diff
@@ -54,6 +62,48 @@ Tags using the new scheme will take the form::
...
+parsing the output::
+
+    import re
+    import subprocess
+
+    def get_musl_major_minor(so: str) -> tuple[int, int] | None:
+        """Detect musl runtime version.
+
+        Returns a two-tuple ``(major, minor)`` that indicates musl
+        library's version, or ``None`` if the given value does not
```
Comment (1ace) [inline pep-0656.rst:79]:
```suggestion
        library's version, or ``None`` if the given libc .so does not
```

:+1: for providing this function btw :)

**Review by 1ace — APPROVED** [review]

File: pep-0656.rst [hunk 3]
```diff
@@ -54,6 +62,48 @@ Tags using the new scheme will take the form::
...
+parsing the output::
+
+    import re
+    import subprocess
+
+    def get_musl_major_minor(so: str) -> tuple[int, int] | None:
+        """Detect musl runtime version.
+
+        Returns a two-tuple ``(major, minor)`` that indicates musl
+        library's version, or ``None`` if the given lib .so does not
```
Comment (1ace) [inline pep-0656.rst:79]: Ah my bad, I made a typo in my suggestion :facepalm: 
```suggestion
        library's version, or ``None`` if the given libc .so does not
```

Comment (uranusjr) [reply pep-0656.rst]: I amended the last commit 🙂

Comment (uranusjr) [reply pep-0656.rst]: I’ve amended the last commit 🙂

File: pep-0656.rst [hunk 4]
```diff
@@ -54,6 +62,48 @@ Tags using the new scheme will take the form::
 
     musllinux_${MUSLMAJOR}_${MUSLMINOR}_${ARCH}
 
+Reading the musl version
+------------------------
+
+The musl version values can be obtains by executing the musl libc
```
Comment (brettcannon) [inline pep-0656.rst:68]:
```suggestion
The musl version values can be obtained by executing the musl libc
```

**Review by brettcannon — APPROVED** [review]

**Pull request merged**